src/app/component/layout/components/search-popup/search-popup.component.ts

```diff
@@ -45,8 +45,18 @@ export class SearchPopupComponent implements OnInit, OnDestroy {
   }
 
   private getNewsAndTips(news, tips): void {
-    (news && news.length > 0) ? (this.isNewsSearchFound = true, this.newsElements = news) : (this.isNewsSearchFound = false);
-    (tips && tips.length > 0) ? (this.isTipsSearchFound = true, this.tipsElements = tips) : (this.isTipsSearchFound = false);
+    if (news.length > 0) {
+      this.isNewsSearchFound = true;
+      this.newsElements = news;
+    } else {
+      this.isNewsSearchFound = false;
+    }
+    if (tips.length > 0) {
+      this.isTipsSearchFound = true;
+      this.tipsElements = tips;
+    } else {
+      this.isTipsSearchFound = false;
+    }
```

Comment:
2 pair of ternary will look better, e.g.
```js
this.isNewsSearchFound = news.length > 0;
...
```

Reply:
fixed